tests/functional/a/async_functions.py

```diff
@@ -39,6 +39,8 @@ async def complex_function(this, function, has, more, arguments, than,
         return than
     try:
         return one
+    except TypeError:
```

Comment:
What was happening here ?

Reply:
Functional tests expected `too-many-branches`.  The try of a try/finally was double-counted before, so here adding an except back corrects the count and keeps the total identical.